pkgs/development/python-modules/hdbscan/default.nix

```diff
@@ -27,6 +27,9 @@ buildPythonPackage rec {
       sha256 = "07d7jdwk0b8kgaqkifd529sarji01j1jiih7cfccc5kxmlb5py9h";
     })
   ];
+  # hdbscan declares a runtime dependency on Cython, but it isn't really necessary.
+  # Patch it out to allow packages depending on hdbscan to build.
+  prePatch = "sed -i~ '/cython/d' requirements.txt";
```

Comment:
please use `postPatch`. That way patches can always apply to the original source.

Reply:
OK, done.